### ISSUE TITLE
Add list management and conversion into JPL doc 

### ISSUE THREAD

**Issue by ssardina** [SWI-Prolog/packages-jpl#41]
While they are compound terms, lists are a special term.

Include a special section for it, explaining how to build them (from a String for example) and how to get a list and convert it back to a String or an array of Terms. For example:

```
System.out.println(Term.toString(sol[i].get("X").toTermArray()));   System.out.println(Term.toString(Util.listToTermArray(sol[i].get("Y"))));
```

Improve the internal documentation as well of those methods.

**Comment by ssardina** [comment]
The JPL doc is hosted here: https://jpl7.org

**Comment by ssardina** [comment]
To convert a list to String this may be the most "faithful" way:

`System.out.println(Arrays.toString(Util.atomListToStringArray(sol[i].get("Y"))));
`
Add this to `Util`

**Comment by ssardina** [comment]
Fixed in 54db229b144b412384c14adc681e69dbbd81d13e
